modules/oe_subscriptions_anonymous/tests/src/Trait/MessageTemplateTrait.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_subscriptions_anonymous\Trait;
+
+/**
+ * Trait to assert messages templates.
+ */
+trait MessageTemplateTrait {
```

Comment:
Why message template? Message templates are for the email sent, not the confirmation messages.

Reply:
Renamed